packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_ValueSpecificationBuilderHelper.ts

```diff
@@ -868,9 +868,10 @@ export function V1_processProperty(
   }
   let inferredType: Type | undefined =
     inferredVariable?.genericType?.value.rawType;
-  if (inferredVariable instanceof AbstractPropertyExpression) {
```

Comment:
Only the subtype condition should be moved to the plugin. The `(inferredVariable instanceof AbstractPropertyExpression` should remain in core.